packages/app-core/src/AppFocus/index.ts

```diff
@@ -17,5 +19,25 @@ export function AppFocusMixin() {
       setFocusedViewId(viewId: string) {
         self.focusedViewId = viewId
       },
+      afterCreate() {
+        addDisposer(
+          self,
+          autorun(() => {
+            const session = self as AbstractSessionModel
+            if (session) {
+              const view = session.views.at(-1)
+              if (
```

Comment:
I believe this makes it impossible now to have a focused view being the subview of a linear synteny view